src/Friend.ts

```diff
@@ -0,0 +1,55 @@
+import Peer from "peerjs";
+import { Message } from "./Message";
+
+export interface IFriend {
+    readonly id: string;
+    readonly displayName: string;
+}
+
+export class Friend {
```

Comment:
This class doesn't really seem to provide us much more than getters and setters, which we could just do with interfaces directly ... something like ...

```ts
export interface IFriendInfo {
  id: string;
  displayName?: string;
}

export interface IFriendConnection extends IFriendInfo {
  conn: Peer.DataConnection;
}
```

Just pass objects complying with that spec around in `Host.js`